osu.Game/Screens/Play/ArgonKeyCounter.cs

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Shapes;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+using osuTK;
+
+namespace osu.Game.Screens.Play
+{
+    public partial class ArgonKeyCounter : KeyCounter
+    {
+        private Circle inputIndicator = null!;
+        private OsuSpriteText countText = null!;
+
+        // These values were taken from Figma
+        private const float line_height = 3;
+        private const float name_font_size = 10;
+        private const float count_font_size = 14;
+
+        // Make things look bigger without using Scale
+        private const float scale_factor = 1.5f;
```

Comment:
Can you explain this a bit? Is it to match figma metrics to game?

Reply:
Basically the Figma values were too small visually, and since I want to keep the Figma values for clarity, I thought that it's best to just scale out the value to make them bigger.